src/docker/Dockerfile

```diff
@@ -52,43 +52,31 @@ RUN apt-get update && apt-get install -y \
     automake autoconf
 
 # Git 2.0
-RUN apt-get install -y software-properties-common python-software-properties
+RUN apt-get update && apt-get install -y \
```

Comment:
@amihaiemil actually you are right, original version was grouped, but at least for me the shorter version is also pretty readable. There are groups, but not named with comments (it's visible what is being added anyway)